code/go/0chain.net/smartcontract/dbs/event/blobber_aggregate.go

```diff
@@ -100,6 +101,8 @@ func (edb *EventDb) calculateBlobberAggregate(gs *globalSnapshot, round, limit,
 	logging.Logger.Debug("blobber_snapshot", zap.Int("total_current_blobbers", len(currentBlobbers)))
 
 	if round <= edb.AggregatePeriod() && len(currentBlobbers) > 0 {
+		logging.Logger.Debug("creating snapshots", zap.Any("currentBlobbers", currentBlobbers))
```

Comment:
@devyetii please check logs to see if this would cause log flush, the currentBlobbers could be a large log block.

Reply:
was for testing, removed.